corehq/apps/data_dictionary/templates/data_dictionary/partials/confirmation_modals.html

```diff
@@ -0,0 +1,194 @@
+                <p>
+                    {% blocktrans %}
+                    This GPS case property is currently being used to store the geolocation for cases.
+                    {% endblocktrans %}
+                </p>
+                <p>
+                    {% blocktrans %}
+                    Deprecating this case property may result in unintended behaviour, and so
+                    it is advised to first change the selected custom case property in
+                    <a href="{{ geospatial_settings_url }}">
+                        geospatial settings
+                    </a>
+                    before deprecating this case property.
+                    {% endblocktrans %}
+                </p>
+
+                <p>
+                    {% blocktrans %}
+                    Would you like to proceed with deprecating this case property?
+                    {% endblocktrans %}
+                </p>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">
+                    {% trans 'Cancel' %}
+                </button>
+                <button id="deprecate-geospatial-prop-btn" type="button" class="btn btn-primary">
+                    {% trans 'Confirm' %}
+                </button>
+            </div>
+        </div>
+    </div>
+</div>
+
+<!-- Modal for deleting case property -->
+<div id="delete-case-prop-modal" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+                <h4 class="modal-title">
+                    {% trans 'Delete Case Property' %}
+                </h4>
+            </div>
+            <div class="modal-body">
+                <p>
+                    {% blocktrans %}
+                        There are currently no cases that store data with this case property, and so it is safe to delete.
+                    {% endblocktrans %}
+                </p>
+                <p>
+                    {% blocktrans %}
+                        Please note however, that the following effects will take place after deleting this case property:
+                    {% endblocktrans %}
+                <ul>
+                    <li>{% trans 'Custom descriptions for the case property will no longer show up in the form builder or case exports.' %}</li>
+                    <li>{% trans 'Any validation rules set up on this case property will no longer be applied on case imports.' %}</li>
+                    {% if request|toggle_enabled:"GEOSPATIAL" %}
+                        <li>{% trans 'This case property will no longer be usable as the geolocation property for the geospatial feature.' %}</li>
```

Comment:
nit: Thinking if this is not needed here. 
1. A geo location property in use can't be deleted
2. if there is data for that case property in cases, it can't be deleted
So, don't think a project would be planning to use an unused case property for geo location. And it is a bit explicit, that if they remove something, they won't be able to use it.

I caution to only keeping really relevant info because having such points could dilute the importance of the other relevant ones.

Reply:
That is a good point. I'll remove this point. @mkangia Do you think the other points are still valid or should be removed as well? If there is no data linked to the deleted case type/property, then I suppose there's no need to have any of these warnings present.

Reply:
I like that you have tried to highlight the overall changes user could see over HQ.
Something similar also done to the modal above for deprecating case types.

One challenge with such *specific* list is to keep it up to date with changes in HQ. For example: if we start using DD in data forwarders, this list would be out of date and would not get updated till someone notices.
I'd recommend to remove it entirely or to keep it generic like

`Please note however, that this case type and its case properties will stop showing on CommCareHQ once deleted, for example as part of suggestions in app builder/reports etc. This will NOT effect any data submission if added for this case type or these case properties in future`

Just wrote that on the fly so please reframe it if you wish to use something like that.

Reply:
That is very true. In terms of maintainability it makes sense to try and keep this as straight-forward and generic as possible. I'm opted to rather remove it entirely, as I don't think we need to go into further detail given that there is no linked case data on the domain.

Reply:
[3c2ac0e](https://github.com/dimagi/commcare-hq/pull/33956/commits/3c2ac0e6c0315307975f23689e82cab830bb8b80)

Reply:
thanks. Should we do that for the deprecating case type modal too? I guess fine to do it in a separate PR. I assume it was added by us only and recently, so not too late to take it back.

Reply:
Given that the deprecating case type modal went through user acceptance testing and was accepted, I would be more hesitant to change it without maybe first getting an Application Engineer's opinion.